pkg/services/provisioning/dashboards/file_reader.go

```diff
@@ -462,6 +463,7 @@ type provisioningMetadata struct {
 }
 
 type dashboardIdentity struct {
+	// Deprecated: use FolderUID instead
```

Comment:
Nit: might be helpful to change the phrasing to use folder UID instead. To me FolderUID sounds like the name of a field but FolderUID doesn't exist as a field. What do you think?